librenms/init.sls

```diff
@@ -122,15 +122,39 @@ librenms_crontab:
 {% else %}
   file.managed:
     - name: /etc/cron.d/librenms
-    - source: {{ librenms.general.home }}/librenms.nonroot.cron
+    - source: {{ librenms.general.app_dir }}/librenms.nonroot.cron
     - require:
       - git: librenms_git
 {% endif %}
 
 librenms_compose_install:
   cmd.run:
-    - name: ./scripts/composer_wrapper.php install --no-dev
+    - name: ./scripts/composer_wrapper.php install --no-dev || ( touch ./trigger_change_in_git_repo; false )
     - runas: {{ librenms.general.user }}
-    - cwd: {{ librenms.general.home }}
+    - cwd: {{ librenms.general.app_dir }}
     - onchanges:
       - git: librenms_git
+      - file: librenms_compose_trigger
+
+librenms_compose_trigger:
+  file.absent:
+    - name: {{ librenms.general.app_dir }}/trigger_change_in_git_repo
+
+librenms_permissions:
```

Comment:
couldn't this be set with salt.states.linux_acl?
https://docs.saltstack.com/en/latest/ref/states/all/salt.states.linux_acl.html

Reply:
@aboe76 I wasn't aware of that state. I'll give it a try. Thanks!

Reply:
This [patch](https://gist.github.com/alxwr/a8186f3fd7409ecad7aa8b73749c3e67) yields

```
----------                                                                
          ID: librenms_rrd_acl
    Function: acl.present        
        Name: /dispensables/librenms/rrd
      Result: False                   
     Comment: An exception occurred in this state: Traceback (most recent call last):                     
                File "/usr/local/lib/python2.7/site-packages/salt/state.py", line 1913, in call          
                  **cdata['kwargs'])           
                File "/usr/local/lib/python2.7/site-packages/salt/loader.py", line 1898, in wrapper
                  return f(*args, **kwargs)                                     
                File "/usr/local/lib/python2.7/site-packages/salt/states/linux_acl.py", line 73, in present                        
                  _current_perms = __current_perms[name].get('defaults', {})
              KeyError: u'/dispensables/librenms/rrd'
     Started: 20:50:06.026572                                                                            
    Duration: 446.473 ms     
     Changes:
```

on FreeBSD 11.2 no matter what combination of ID and name I try.
Like in `permissions.sh` I have to exempt FreeBSD from setting the ACL. (32571f18110386f1f13c9e5375d20ec951feb671)

`states.linux_acl` works fine on Ubuntu 18.04 though.